src/urg_c_wrapper.cpp

```diff
@@ -406,6 +406,120 @@ bool URGCWrapper::getAR00Status(URGStatus& status)
   return true;
 }
 
+bool URGCWrapper::getDL00Status(UrgDetectionReport& report)
+{
+  // Construct and write DL00 command.
+  std::string str_cmd;
+  str_cmd += 0x02;  // STX
+  str_cmd.append("000EDL005BCB");  // DL00 cmd with length and checksum.
+  str_cmd += 0x03;  // ETX
```

Comment:
Doesn't have to be now, but if there is any more of this kind of thing, let's move this stuff out to a helper function.